ci/travis/after_success-release-linux.sh

```diff
@@ -32,11 +34,12 @@ sudo pip install twine || exit
 echo "Creating distribution files..."
 # This release build creates the source distribution. All other release builds
 # should not.
-python setup.py sdist bdist || exit
+python setup.py sdist bdist bdist_wheel || exit
```

Comment:
Why use twine below in lieu of `python setup.py sdist bdist bdist_wheel upload` ?